mediapipe/tasks/ios/core/sources/MPPTaskResult.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The MediaPipe Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * MediaPipe Tasks options base class. Any MediaPipe task-specific options class should extend
+ * this class.
+ */
+NS_SWIFT_NAME(TaskResult)
+@interface MPPTaskResult : NSObject <NSCopying>
```

Comment:
This class seems to exist only in iOS. Is this needed or can we follow the other implementations more closely?

Reply:
Following the Java implementation here.